spk/synocli-net/Makefile

```
@@ -1,10 +1,10 @@
 SPK_NAME = synocli-net
 SPK_VERS = 1.3
```

Please also increase package version itself
```suggestion
SPK_VERS = 1.4
```